### ISSUE TITLE
LVM mode: device manager does not parse volume size correctly

### ISSUE THREAD

**Issue by okartau** [intel/pmem-CSI#134]
When LVM-mode device manager lists Volumes, it incorrectly gets Size=0:
(Parse output added for debug)
```
Executing: lvs --noheadings -o lv_name,lv_path,lv_size --units B ndbus0region0fsdax
Output:   3616382b-1989-11e9-9797-525400a0ae34 /dev/ndbus0region0fsdax/3616382b-1989-11e9-9797-525400a0ae34 1073741824B
Parse: Name:3616382b-1989-11e9-9797-525400a0ae34 Path:/dev/ndbus0region0fsdax/3616382b-1989-11e9-9797-525400a0ae34 Size:0
```

Currently this causes no detectable problems as Size is not used for anything, but should be fixed.

**Comment by okartau** [comment]
The reason is the appended 'B' character to size, ParseUint fails to parse such value.
Problem can be avoided by adding --nosuffix to lvs options.

**Comment by okartau** [comment]
fix to this issue exists now as a commit in #133, as this was the tree I kept working. It does not logically belong there as #133 is about direct mode and fix is about LVM, we could consider extracting fix from there and creating separate PR for LVM-mode fix(es)

**Comment by okartau** [comment]
did what I proposed in previous comment. Fix to current issue is now part of #138

**Comment by okartau** [comment]
closing as fix was merged
